test/k8s-integration/config/test-config.yaml

```diff
@@ -0,0 +1,32 @@
+StorageClass:
+  FromFile: /usr/local/google/home/saikatroyc/go/src/sigs.k8s.io/gcp-filestore-csi-driver/test/k8s-integration/config/fs-sc-basic-hdd.yaml
+
+SnapshotClass:
+  FromFile: /usr/local/google/home/saikatroyc/go/src/sigs.k8s.io/gcp-filestore-csi-driver/test/k8s-integration/config/fs-backup-volumesnapshotclass.yaml
+
+DriverInfo:
+  Name: csi-gcpfs-fs-sc-basic-hdd
+  SupportedFsType:
+    nfs:
```

Comment:
do you actually want an explicit nfs option and not leave it empty? What if filestore supports smb in the future? I think it should be possible the same PV could be used for smb or nfs.

Reply:
Leaving SupportedFsType as empty now